.travis.yml

```diff
@@ -2,16 +2,16 @@ sudo: false
 language: ruby
 
 rvm:
-  - 2.5.0
-  - 2.4.3
-  - 2.3.6
-  - 2.2.9
-  - 2.1.10
```

Comment:
For some reason I couldn't figure out, rubocop behaves differently on 2.1: 

```
Offenses:
spec/models/letter_opener_web/letter_spec.rb:17:1: C: Layout/ClosingHeredocIndentation: MAIL is not aligned with <<-MAIL.
    MAIL
^^^^^^^^
```

So I decided to drop the 2.1 testing and replace it by 2.6.